pkg/ssh/ssh.go

```diff
@@ -105,6 +105,59 @@ func (cfg *Config) addSSHFlag(s string) error {
 	return nil
 }
 
+func (cfg *Config) ApplyEnvironment(grafanaPDCEnvVarPrefix string) {
+	sshKeyFileEnvVar, ok := os.LookupEnv(fmt.Sprintf("%v_SSH_KEY_FILE", grafanaPDCEnvVarPrefix))
+	if ok && (cfg.KeyFile == DefaultConfig().KeyFile) {
+		cfg.KeyFile = sshKeyFileEnvVar
+	}
+
+	// Using the same logic as the `ssh.Config` definition, clamping to max of 3
+	sshLogLevelSelectEnvVar, ok := os.LookupEnv(fmt.Sprintf("%v_SSH_LOG_LEVEL", grafanaPDCEnvVarPrefix))
+	parsedSSHLogLevelSelectEnvVar, parseIntErr := strconv.Atoi(sshLogLevelSelectEnvVar)
+	if ok &&
+		parseIntErr == nil &&
+		(cfg.LogLevel > 3 || cfg.LogLevel < 1) &&
+		(parsedSSHLogLevelSelectEnvVar < 3 && parsedSSHLogLevelSelectEnvVar >= 0) {
+		cfg.LogLevel = parsedSSHLogLevelSelectEnvVar
+	}
+
+	skipSSHValidationEnvVar, ok := os.LookupEnv(fmt.Sprintf("%v_SSH_SKIP_SSH_VALIDATION", grafanaPDCEnvVarPrefix))
+	parsedSSHValidationEnvVar, parseBoolErr := strconv.ParseBool(skipSSHValidationEnvVar)
+	if ok && parseBoolErr == nil && !cfg.SkipSSHValidation {
+		cfg.SkipSSHValidation = parsedSSHValidationEnvVar
+	}
+
+	// We're actually going to append the values provided here to the already-parsed SSHFlags array
+	// As the argument can be provided more than once, this feels like the most compatible option
+	additionalSSHFlagsEnvVar, ok := os.LookupEnv(fmt.Sprintf("%v_SSH_ADDITIONAL_SSHFLAGS", grafanaPDCEnvVarPrefix))
```

Comment:
```suggestion
	additionalSSHFlagsEnvVar, ok := os.LookupEnv(fmt.Sprintf("%v_SSH_ADDITIONAL_SSH_FLAGS", grafanaPDCEnvVarPrefix))
```

As it is, this is inconsistent with the README. I'm OK with the use of `ADDITIONAL`